wallet-new/src/Cardano/Wallet/API.hs

```diff
@@ -24,13 +27,37 @@ import qualified Cardano.Wallet.API.V1 as V1
 -- * 'Cardano.Wallet.Server' contains the main server;
 -- * 'Cardano.Wallet.API.V0.Handlers' contains all the @Handler@s serving the V0 API;
 -- * 'Cardano.Wallet.API.V1.Handlers' contains all the @Handler@s serving the V1 API;
+-- * 'Cardano.Wallet.API.Dev.Handlers' contains all the @Handler@s serving the Dev API;
 --
+-- type WalletAPI
+--     =     WalletPublicAPI
+--     :<|>  WalletDevAPI
+
+type V0_API
+     = "api" :> Tags '["V0 (Deprecated)"]
+             :> V0.API
+
+type V1_API
+     = "api" :> "v1"
+             :> Tags '["V1"]
+             :> V1.API
+
+type DEV_API
+     = "api" :> "development"
+             :> Tags '["Development"]
+             :> Dev.API
+
 type WalletAPI
-    =    "api" :> Tags '["V0 (Deprecated)"]
-               :> V0.API
-    :<|> "api" :> "v1"
-               :> Tags '["V1"]
-               :> V1.API
+    =     V0_API
+    :<|>  V1_API
+    :<|>  DEV_API
+
+type WalletSwaggerApi
+    =     V0_API
+    :<|>  V1_API
 
 walletAPI :: Proxy WalletAPI
 walletAPI = Proxy
+
+swaggerWalletApi :: Proxy WalletSwaggerApi
+swaggerWalletApi = Proxy
```

Comment:
This shouldn't belong here. There is a reason why `Swagger` stuff is in `server`, and that's because I've tried to separate the documentation concerns (which lives in executable code) vs purely API concerns (which live in `src`) 😉

Reply:
We might not need it anymore...